### ISSUE TITLE
road builder to get longest road

### ISSUE THREAD

**Issue by dannyharding10** [thyer/CS340winter2016#691]
if you use road builder to achieve the longest road, it doesn't give you the card or the 2 points.  To reproduce: 

make 3 or 4 roads. 
use road builder dev card to get longest road(on mine I connected two roads that were already there. 
see that you don't have the longest road card or 2 extra points.

**Comment by eaglewyng** [comment]
This _may_ be the same bug as #695, where the road builder card isn't getting received by the server.

**Comment by eaglewyng** [comment]
Confirmed that this is the same issue as #695. Marking as duplicate.

**Comment by dannyharding10** [comment]
I don't think so, because on #695 all the roads disappeared after ending the turn, but with this the roads stayed there.

**Comment by eaglewyng** [comment]
hm ok. I'll remove the duplicate tag and retry this.

**Comment by sadler-j** [comment]
There shouldn't be an issue with road builder placing roads. This is either from the road not getting placed, or there were intersections in the road which seem to be causing issues.
